modules/machine-user-infra-machines-iso.adoc

```diff
@@ -19,17 +19,39 @@ You can create more {op-system-first} compute machines for your bare metal clust
 ** Burn the ISO image to a disk and boot it directly.
 ** Use ISO redirection with a LOM interface.
 
-. After the instance boots, press the `TAB` or `E` key to edit the kernel command line.
-. Add the parameters to the kernel command line:
+. Boot the {op-system} ISO image without specifying any options or interrupting the live boot sequence. Wait for the installer to boot into a shell prompt in the {op-system} live environment.
++
+[NOTE]
+====
+It is possible to interrupt the {op-system} installation boot process to add kernel arguments. However, for this ISO procedure you should use the `coreos-installer` command as outlined in the following steps, instead of adding kernel arguments.
```

Comment:
Just a suggestion, upto you.
```suggestion
You can interrupt the {op-system} installation boot process to add kernel arguments. However, for this ISO procedure you should use the `coreos-installer` command as outlined in the following steps, instead of adding kernel arguments.
```